test/blackbox/common/BlackboxTests.hpp

```diff
@@ -139,6 +139,9 @@ void default_send_print(
 std::list<HelloWorld> default_helloworld_data_generator(
         size_t max = 0);
 
+std::list<HelloWorld> default_large_helloworld_data_generator(
+        size_t max = 0);
+
```

Comment:
We don't need this. We can use `Data1mb` as data type, and use `default_data300kb_data_generator`

Reply:
I have thought of using it but the `send()` and `expected_data()` functions don't support this datatype as they are

Reply:
You would also need to change the TypeSupport when instantiating `RTPSWithRegistrationWriter` and `RTPSWithRegistrationReader`:

`RTPSWithRegistrationWriter<Data1mbPubSubType> writer(TEST_TOPIC_NAME);`
`RTPSWithRegistrationReader<Data1mbPubSubType> reader(TEST_TOPIC_NAME);`